src/getCurrentPosition/getCurrentPosition.types.ts

```diff
@@ -12,22 +12,22 @@ export type GeolocationSuccessCallback = (position: {
 }) => void;
 
 export type GeolocationErrorCallback = (error: {
-    code: GeolocationErrorCode;
+    code: (typeof GeolocationErrorCode)[keyof typeof GeolocationErrorCode];
     message: string;
-    PERMISSION_DENIED: GeolocationErrorCode.PERMISSION_DENIED;
-    POSITION_UNAVAILABLE: GeolocationErrorCode.POSITION_UNAVAILABLE;
-    TIMEOUT: GeolocationErrorCode.TIMEOUT;
+    PERMISSION_DENIED: typeof GeolocationErrorCode.PERMISSION_DENIED;
+    POSITION_UNAVAILABLE: typeof GeolocationErrorCode.POSITION_UNAVAILABLE;
+    TIMEOUT: typeof GeolocationErrorCode.TIMEOUT;
```

Comment:
sorry we know we're in hurry to get this one merged, but one more question 🙇 

Does the error callback always called with

```ts
{
    ...
    PERMISSION_DENIED: 1,
    POSITION_UNAVAILABLE: 2;
    TIMEOUT: 3;
    ...
}
```

Reply:
Yeah, it's interesting - I assume they did it like that so you could compare the output with one of the selected options

Reply:
ah okay thank you for confirmation 🙇